### PR TITLE
[ERC20] test bad code

### PR DESCRIPTION
**Pull request by nrryuya**
- [The original `test_bad_deposit`](https://github.com/ethereum/vyper/blob/97ed5fec89947d41fd91f5190c4478ffcadbb9fb/tests/examples/tokens/ERC20_solidity_compatible/test/erc20_tests_1.py#L370) seems wrong

### CODE REVIEW

File: tests/erc20/test_erc20_mintable_burnable.py [hunk 1]
```diff
@@ -299,3 +312,36 @@ def test_raw_logs(c, w3, get_log_args):
 
 def test_failed_send_in_withdraw(c, w3):
     pass
+
+
+def test_bad_transfer(c_bad, w3, assert_tx_failed):
+    # Ensure transfer fails if it would otherwise overflow balance when totalSupply is corrupted
+    minter, a1, a2 = w3.eth.accounts[0:3]
+    c_bad.mint(a1, MAX_UINT256, transact={'from': minter})
+    c_bad.mint(a2, 1, transact={'from': minter})
```
Comment (yudetamago) [inline tests/erc20/test_erc20_mintable_burnable.py]: (Just to confirm) This line checks that total_supply doesn't increase?

Comment (nrryuya) [reply tests/erc20/test_erc20_mintable_burnable.py]: The purpose of `bad_code` is to check the `balances` storage variable correctly detects overflow and reverts.
In `transfer`,  `transferFrom` and `burn`, `totalSupply` is increased/decreased before `balances` so we can't check that in the correct code. That's why we remove these codes as `bad_code` .

File: .python-version [hunk 2]
```diff
@@ -0,0 +1 @@
+fvyper
```
Comment (yudetamago) [inline .python-version]: This file is added unintentionally?

File: k/.python-version [hunk 3]
```diff
@@ -0,0 +1 @@
+system
```
Comment (yudetamago) [inline k/.python-version]: This file is added unintentionally?

File: tests/erc20/test_erc20_mintable_burnable.py (hunk 1 above)
Comment (yudetamago) [reply tests/erc20/test_erc20_mintable_burnable.py]: got it!

**Comment by yudetamago** [comment]
LGTM

**Pull request merged**